platform/android/export/export.cpp

```diff
@@ -1831,6 +1845,7 @@ void register_android_exporter() {
 	//EDITOR_DEF("android/release_keystore","");
 	//EDITOR_DEF("android/release_username","");
 	//EditorSettings::get_singleton()->add_property_hint(PropertyInfo(Variant::STRING,"android/release_keystore",PROPERTY_HINT_GLOBAL_FILE,"*.keystore"));
+	EDITOR_DEF("android/force_system_user", false);
```

Comment:
this `android/force_system_user` default value is different from L1692.
shouldn't it have the same default value?